### PR TITLE
LPD-16409 Add test to delete entry when object definition has relationship with a draft object relationship

### PR DESCRIPTION
**Pull request by thallescezar**
Related issue: https://liferay.atlassian.net/browse/LPD-16409

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -2139,6 +2147,39 @@ public void testDeleteObjectEntry() throws Exception {
...
+		// Delete object entry with an object definition that is related to a
+		// draft object definition
+
+		_objectDefinition = _publishCustomObjectDefinition(
+			false,
+			Arrays.asList(
+				ObjectFieldUtil.createObjectField(
+					ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+					ObjectFieldConstants.DB_TYPE_STRING,
+					RandomTestUtil.randomString(), StringUtil.randomId())));
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2163]: Hey @thallescezar this variable is being used to delete another object definition that was created on setUp, once you create another object definition and change the variables to yours it'll delete the last one and will keep the one created on set up in the database, but actually I believe both object definitions are being deleted after test run, if so, please define the draft object definition and this new one here only in this scope.

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by thallescezar** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 2]
```diff
@@ -2139,6 +2138,49 @@ public void testDeleteObjectEntry() throws Exception {
...
+		ObjectDefinition draftObjectDefinition =
+			ObjectDefinitionTestUtil.addCustomObjectDefinition(
+				false, _objectDefinitionLocalService,
+				Arrays.asList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING,
+						RandomTestUtil.randomString(), StringUtil.randomId())));
+
+		ObjectDefinition objectDefinitionNew = _publishCustomObjectDefinition(
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2157]: Hey @thallescezar rename `objectDefinitionNew` to `publishedObjectDefinition`,

after this you can forward to Brian, thanks!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by thallescezar** [comment]
ci:test:relevant

**Comment by thallescezar** [comment]
ci:test:object

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by thallescezar** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by thallescezar** [comment]
ci:forward:force

**Pull request closed without merging**